### PR TITLE
Fix ContactsArrayMessage and add getBusinessProfile

### PR DESCRIPTION
**Pull request by BochilGaming**

### CODE REVIEW

**Review by PurpShell — APPROVED** [review]
lgtm

File: src/Types/index.ts [hunk 1]
```diff
@@ -67,4 +67,17 @@ export type WABusinessProfile = {
...
+    description: string,
+    website?: string,
+    email?: string,
+    category: string,
+    business_hours: {
+        timezone: string,
+        business_config: WABusinessHoursConfig[]
+    }
+}
+
```
Comment (adiwajshing) [inline src/Types/index.ts:82]: would be better to use the same structure for legacy & MD

File: src/Socket/chats.ts [hunk 2]
```diff
@@ -158,6 +158,49 @@ export const makeChatsSocket = (config: SocketConfig) => {
             ]
         })
     }
+    const getBusinessProfile = async (jid: string): Promise<WABusinessProfileMD | {}> => {
```
Comment (adiwajshing) [inline src/Socket/chats.ts:161]: Should just return undefined -- not an empty object

**Comment by BochilGaming** [comment]
ok done

**Comment by adiwajshing** [comment]
@BochilGaming please revert all formatting you've done -- stick to only changing code that needs to be changed

**Comment by BochilGaming** [comment]
> @BochilGaming please revert all formatting you've done -- stick to only changing code that needs to be changed

sorry for that

**Pull request merged**